lib/server/controllers/logs.controller.js

```diff
@@ -25,14 +25,32 @@ const { repositories: { LogRepository } } = require('../../database');
  *                        next middleware function.
  * @returns {undefined}
```

Comment:
does it?

Reply:
Are you referring to the following:
```diff
-      return response.status(400).json({
+      response.status(400).json({
            errors,
        });
+     return;
```

Reply:
line 42 to 44,  should these be documented?